resources/guis.yml

```diff
@@ -399,4 +399,15 @@ guis:
   price: Free
   license: Vim
   trend_name: Vim fugitive
-  order: 12
+  order: 16
+- name: MeGit (based on EGit)
+  url: https://github.com/eclipsesource/megit
+  image_tag: guis/megit@2x.png
+  platforms:
+  - Windows
+  - Mac
+  - Linux
+  price: Free
+  license: EPL2.0
+  trend_name: egit
+  order: 9
```

Comment:
I was surprised by the high ranking here for a new tool. But it's because the trend name is `egit`, not `megit`. From what I understand this is basically a packaging of the same `egit` interface, so in a sense it's fair. But it also feels a little like cheating. :wink: I dunno. I guess this is just about ordering things and not any kind of important cutoff, so perhaps it makes sense to give it a chance to get some traction. We'll end up running the sort script again later, and we can perhaps revisit it then.